hyperframe/frame.py

```diff
@@ -316,8 +323,14 @@ def serialize_body(self):
         return self.serialize_priority_data()
 
     def parse_body(self, data):
+        if len(data) > 5:
+            raise InvalidFrameError(
+                "PRIORITY must have 5 byte body: actual length %s." %
+                len(data)
+            )
+
         self.parse_priority_data(data)
-        self.body_len = len(data)
+        self.body_len = 5
```

Comment:
Could you put the 5 as a CONSTANT at the top of the file? Seems to match the style used previously. Also allows the commenting convention.

Reply:
Which style does it match?
All fixed-length frames have a "self.body_len = XYZ" statement:
RstStreamFrame, PingFrame, WindowUpdateFrame

Reply:
Oh yea, fair enough.